src/main/java/org/apache/commons/math4/util/FastMath.java

```diff
@@ -1810,15 +1829,17 @@ private static double polyCosine(double x) {
      *  @param xb extra bits for x (may be 0.0)
      *  @return sin(xa + xb)
      */
-    private static double sinQ(double xa, double xb) {
-        int idx = (int) ((xa * 8.0) + 0.5);
-        final double epsilon = xa - EIGHTHS[idx]; //idx*0.125;
+static double pi_half = PI / 2;
```

Comment:
This is out of place here. If it is required, move from between the javadoc and the method to its own proper place and add an explanation. It seems to be needed for `@LessThan("pi_half")`, but I am not sure if we should be using `@LessThan` for doubles.

Reply:
Should I remove the `@LessThan("pi_half")` annotations and suppress the warnings in the relevant places?

Reply:
Ok.